cmd/otelcol/Dockerfile

```
@@ -1,16 +1,36 @@
-FROM alpine:3.12 as certs
+ARG SMART_AGENT_RELEASE=v5.9.1
+ARG ALPINE_VERSION=3.12
+
+FROM alpine:${ALPINE_VERSION} as certs
 RUN apk --update add ca-certificates
 
-FROM alpine:3.12 AS otelcol
+FROM alpine:${ALPINE_VERSION} AS otelcol
 COPY otelcol /
 # Note that this shouldn't be necessary, but in some cases the file seems to be
 # copied with the execute bit lost (see #1317)
 RUN chmod 755 /otelcol
 
+FROM alpine:${ALPINE_VERSION} AS smartagent
+ARG SMART_AGENT_RELEASE
+RUN mkdir -p "$(dirname "/usr/lib/splunk-otel-collector/agent-bundle")" && \
+    SMART_AGENT=signalfx-agent-${SMART_AGENT_RELEASE//"v"/""}.tar.gz && \
```

```suggestion
    SMART_AGENT=signalfx-agent-${SMART_AGENT_RELEASE#v}.tar.gz && \
```
Just a nit and probably wouldn't ever be an issue, but maybe best to just strip the leading `v` instead of replacing all occurrences.